sc-kpm/sc-python/services/http_api/auth/private.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpgIBAAKCAQEA5d5ug4guxO6e0Ig+/+hlpwQAoMbxGU8K3TFB67UbK9BXFj9m
+5jv/b2c8VcWySgtVy5a2D5gzRsYuzjBoMCYWPVyTCo5jUCXbKU3JZNNS9BddX2l+
+KUe3LSubxkU71lnYVQNmGJg+CC173oVgwVSrFN7BuJ8abh/LzpEbZPVMa5S2PrM0
+S33k+tjnn+vG2axmw0simPoD8VdR1ajWzFtwJyUXvARkSUQ+tHouao+iuRi/mQdU
+H5zLsdVBrAXESplfSThc0P+cz2n2XiKKDLpxmXNwf+NbuAqNTdCJVjORZYML0tNm
+wquLG9gJFcQCOCUzl39MUOuvoNxWTGx6DuuubQIDAQABAoIBAQCQZwUh1+EZkZVv
+Bsgc1w9Utiz200MOVRtMDO+OdP0WrxCmRzoLFFHI2B+M+l6gKTLCbHD+puw/X6xz
+lvE8nhCGh6tPjFdTOBTPg7AqiJ6p/CuKDqlUoVqvI5/+E+wdId/GuBXhCXKejIQu
+K7e2e8vllnI32qst1wefQoTlDTvusBJb4q9rJKKIs7x8wcaFiACmTgCvMEK9l7HM
+iMSzL91rXYuf/DrZXkwogFV5RwEhvPgdw6oMZvdLZE79r6xyPDXv53CxbG0SPqzS
+a9DEADpBy4sCMbVKIXoeMXcft/ETC8zm5tmSjsir8q0S+uVqlvewdBlMQnn+f/Qy
+S/JMh83BAoGBAPeU/G2SE0e+f06NwnC1HCWUrQYLXmpSpAppKmqKvtlt2IC3Od+f
+Iqa1Kn6C3Z0n/vOA323s8oGJ5WeFC2aWU2Tnd+yTYfheS0vyDkH5tVV8za7ejoXq
+uZiIt+J+JlmKSfC2VIFmSMcK9/9jtDDwWe0SyGO0TFKiaoJuudVDrvq1AoGBAO2v
+RDfLr1nvV9ZHb8rS7cL2/HvoJnN9x2RmGRwOJaHiSHR2GTwDuiobEN8wp+Vm7clK
+qSmjpGJkWsn8NKQ0r5XldJLbmUyNAN/RC4s+WqUI5mUbMIynLpSwh4M7THOYCXzk
+ojY/937kGKdaFofV0EK6nU3Q0Z3UlaburCLIUF/ZAoGBAJ2R1q4tn9Oza/akxjFR
+Mt7AJvaH2ck1W228zOAN+bUL8hTJiC3IbBdJiggGZeEViW+Uf57IhPbounYN+NmL
+pvu7TVnl2x5ldOqa3XQj5sLHsWgwAOk+IGzx5oZh3ypxlul6QYIxhPQZCQkjfZUA
+A1uc7qh2IXUeJUdqDR+r+ftZAoGBALYteGQrwqN0SCb9ajpTodml5bAzKAWtin7x
+op1WWpHSFybsvjq5wJ81T6FCce3rDqcSisZXn0+rC8t1fAPAPjTuQoOtvbpm+RyT
+5f08Nc35HF6ykGu+WH2vMK7tbJPNHLiVpZ9fOBwSSLM84bgLsizzTE6wl3oZzvq8
+RUQDtVdBAoGBAK0gsEppE/zXNbgPWiLCdDJs9PJW9m4pLIRZgQBbPDvvVWIgH+BA
+bX1eFvq/e7kvVpQgDGAlrNyJmuBfPcFOz3hNbVsT94gcafAeH6/hdP3k0Xm+BpKB
+R5B0CLsOZGs688H5xi405tTZrOBHxiBwQL4veRHKDuLcfb+3EyFOvzzH
+-----END RSA PRIVATE KEY-----
```

Comment:
As I remember, better to store such keys in .env files and not push them in repo.